x-pack/dockerlogbeat/magefile.go

```diff
@@ -147,11 +146,14 @@ func Package() {
 // Release builds a "release" tarball that can be used later with `docker plugin create`
 func Release() error {
 	mg.Deps(Build)
-	name, err := getPluginName()
```

Comment:
`name` is used to format the bash script, I think it will fail now?

Reply:
`name` is defined earlier in the magefile, if that's what you're asking.

Reply:
ouch, sorry I missed that!